### ISSUE TITLE
Add a private message system

### ISSUE THREAD

**Issue by Abwasserrohr** [OdyseeTeam/odysee-frontend#907]
**Is your feature request related to a problem? Please describe.**
While it's great to have communicaation over Telegram or Discord, a user [suggested](https://odysee.com/@OdyseeDE:1c44ca079e0e3a824881184fdffcf1864cb2649e?view=discussion&lc=050a14d90fdb0e5c9e25aa8a522bdf4eecc506f3f750ec250aa84910d43fe35f) it would be cool to be able to send private messages on the platform. YouTube had it for some time, but then they disabled it.

**Comment by tzarebczan** [comment]
I don't think we plan to add this anytime soon, and when we do get to it, this issue would be opened on the commetron repo first. Will move there..
